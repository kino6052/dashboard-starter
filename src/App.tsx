import React from 'react';
import { PersistentDrawer } from './Components/AppBar/PersistenDrawer/PersistentDrawer';

export function App() {
  return (
    <div className="App">
      <PersistentDrawer />
    </div>
  );
}
