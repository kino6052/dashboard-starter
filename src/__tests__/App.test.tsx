import React from 'react';
import { App } from "../App";
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('App.ts', () => {
    it('Should correctly render app', () => {
        const wrapper = shallow(<App />);
        expect(wrapper.length).toBe(1);
        expect(wrapper).toMatchSnapshot();
    })
})