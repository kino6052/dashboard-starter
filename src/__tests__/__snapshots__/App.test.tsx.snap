// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.ts Should correctly render app 1`] = `ShallowWrapper {}`;
